content/blog/2022/10/2022-10-14-jenkins-newsletter.adoc

```diff
@@ -0,0 +1,156 @@
+---
+layout: post
+title: "Jenkins September 2022 Newsletter"
+tags:
+- jenkins
+- newsletter
+- community
+- contribute
+authors:
+- alyssat
+opengraph:
+  image: /images/post-images/2022-10-14-jenkins-newsletter/image1.png
+---
+
+== September 2022
+
+image:/images/post-images/2022-10-14-jenkins-newsletter/image1.png[image,width=289,height=175] Monthly Newsletter
+
+Welcome to the Jenkins Newsletter!
+This is a compilation of progress within the project, highlighted by Jenkins Special Interest Groups (SIGs) for the month of September.
+
+Happy reading!
+
+== Security Update
+
+* Two security advisories during September with "special" nature for Jenkins core
+** One was only impacting very specifically configured instance
+** The other was only impacting three successive weekly releases
+
+== Infrastructure Update 
+* Dropping JDK8 from all agents and controllers processes with the 2.361.1 September LTS
```

Comment:
```suggestion

* Dropping JDK8 from all agents and controllers processes with the 2.361.1 September LTS
```

nitpick: Wether you are using asciidoctor or Markdown, surrounding blocks with an (only 1) empty line is considered a best practise:

- Clarity of the source (when reading/reviewing)
- Help different parsers (IDEs, previews in GitHub, CLIs)